docs/resources/gateway.md

```diff
@@ -186,7 +186,7 @@ resource "valtix_gateway" "gcp-gw" {
 * `packet_capture_profile` - (Optional) Packet Profile ID *(e.g. valtix_profile_packet_capture.pcap1.profile_id)*
 * `diagnostics_profile` - (Optional) Diagnostics Profile ID *(e.g. valtix_profile_diagnostics.diag1.profile_id)*
 * `settings` - (Optional) Gateway settings. This block can be repeated multiple times. Please check [this section](#gateway-settings) for the structure. 
-* `instance_details` - (Required) This block can be repeated multiple times for deploying Gateway instances in multiple zones. Look below for the [structure](#instance-details) of this block. Atleast 1 block must be provided.
+* `instance_details` - (Required) This block can be repeated multiple times for deploying Gateway instances in multiple zones. Look below for the [structure](#instance-details) of this block. At least 1 block must be provided.

```

Comment:
can we say that "atleast 1 block must be provided if the vpc_id is not a service_vpc". And if its service vpc, then this must/need not be provided